### PR TITLE
frontend/aopp: blocking dialog when verifying address

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/components/aopp/verifyaddress.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+interface VerifyAddressProps {
+    accountCode: string;
+    address: string;
+    addressID: string;
+}
+
+type Props = VerifyAddressProps & TranslateProps;
+
+class VerifyAddress extends Component<Props, State> {
```
Comment (thisconnect) [inline frontends/web/src/components/aopp/verifyaddress.tsx:35]: Nit: either

```js
public readonly state: State = {
        verifying: false,
};
```

or make `verifying` optional in the State interface

File: frontends/web/src/components/aopp/aopp.tsx [hunk 2]
```diff
@@ -24,7 +24,8 @@ import { Fullscreen, FullscreenHeader, FullscreenContent, FullscreenButtons } fr
 import { Message } from '../message/message';
 import { Button, Field, Input, Label, Select } from '../forms';
 import { CopyableInput } from '../copy/Copy';
-import { ArrowDown, BitBox02Stylized, Cancel, Checked } from '../icon';
+import { ArrowDown, BitBox02Stylized,Cancel, Checked } from '../icon';
```
Comment (thisconnect) [inline frontends/web/src/components/aopp/aopp.tsx:27]: space after comma

**Review by thisconnect — APPROVED** [review]
LGTM with small nit

File: frontends/web/src/components/aopp/verifyaddress.tsx (hunk 1 above)
Comment (benma) [reply frontends/web/src/components/aopp/verifyaddress.tsx:35]: why? it is false by default.

Comment (thisconnect) [reply frontends/web/src/components/aopp/verifyaddress.tsx:35]: Where is it false by default?
It looks `undefined`

File: frontends/web/src/components/aopp/aopp.tsx (hunk 2 above)
Comment (thisconnect) [reply frontends/web/src/components/aopp/aopp.tsx:27]: @benma ☝️

**Pull request merged**